src/Selenium2Library/locators/elementfinder.py

```diff
@@ -89,6 +90,10 @@ def _find_by_tag_name(self, browser, criteria, tag, constraints):
         return self._filter_elements(
             browser.find_elements_by_tag_name(criteria),
             tag, constraints)
+            
+    def _find_by_sc_locator(self, browser, criteria, tag, constraints):
+        js = "return isc.AutoTest.getElement('%s')" % criteria.replace("'", "\\'")
```

Comment:
I think they are just handling single quotation marks embedded in a string using single quotes as the start and termination characters so the JS statement is syntactically OK. jQuery selector is very similar and does the same thing. The double backslash is to escape the backslash which is a special char in Python.
Leaning toothpick syndrome!
Without replace (bad):
`return isc.AutoTest.getElement('//ListGrid[ID='filterGrid']')`
With replace (good):
`return isc.AutoTest.getElement('//ListGrid[ID=\'filterGrid\']')`